File: src/main/scala/study/HelloWorld.scala
package study

import java.io.File
import scala.util.control.Breaks._
import java.io._

/**
  * Created by wangfx on 2017/1/14.
  */
object HelloWorld extends App{
    println("hello world")

  val fileHere = (new File(".")).listFiles
  for (file <- fileHere
      if file.isFile
      if file.getName.endsWith(".scala"))
    println(file)


  val firstArg = if (args.length > 0) args(0) else ""
  firstArg match  {
    case "salt" => println("pepper")
    case "chips" => println("salsa")
    case "eggs" => println("bacon")
    case  _ => print("huh?")
  }

  val in = new BufferedReader(new InputStreamReader(System.in))
  breakable {
    while(true) {
      println("? ")
      if (in.readLine() == "") break()
    }
  }


}

File: src/main/java/org/stephen/HelloWorld.java
package org.stephen;

import java.sql.Timestamp;

/**
 * Created by wangfx on 2017/1/12.
 */
public class HelloWorld {
    public static void main(String[] args) {
        System.out.println("hello new world!");
//        Hello hello = new Hello();
//        hello.sayHello("scala");


        Timestamp t = new Timestamp(System.currentTimeMillis());
        System.out.println(t);
    }
}
